48083 Field Level View of Audit History data for any specific field u…

tyndmar opened the pull request:
…sing a hot key

See Zoho comment for details

Commit:
48083 Field Level View of Audit History data for any specific field using a hot key
See Zoho comment for details

brad-advantzware reviewed (changes requested):
After discussion, I would really like to avoid introducing another shared var.  I generally agree that we should really be pulling the session company from the Session.p persistent.

Commit:
48083 Field Level View of Audit History data for any specific field using a hot key
Removed shared variable; replaced with procedure call to system/session.p in methods/ctrl-f.i

Commit:
48083 Field Level View of Audit History data for any specific field using a hot key
forgot to include changed file
